### PR TITLE
AddToolKits

### PR DESCRIPTION
**Pull request by rmorgens**
Adding breakouts of the various Tool Kit types with their respective modifiers

### CODE REVIEW

File: src/items/equipment/tool_kit_akinzi_resources_catalyst_rig.json [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    },
+    "equippable": false,
+    "equipped": false,
+    "hands": 2,
+    "identified": true,
+    "level": 1,
+    "modifiers": [
+      {
+        "_id": "1d2a51d3-79fe-47ff-b79a-26deff26a5bf",
+        "name": "Catalyst Rig - LS",
```
Comment (LebombJames) [inline src/items/equipment/tool_kit_akinzi_resources_catalyst_rig.json:68]: These modifiers to skill ranks don't need to exist; for crafting, all skill ranks do is determine whether you can craft something or not. There is no check involved

File: src/items/equipment/tool_kit_akinzi_resources_catalyst_rig.json [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    },
+    "description": {
+      "chat": "",
+      "gmnotes": "",
+      "short": "",
+      "unidentified": "",
+      "value": "<p>Akinzi Resources, a startup company operating out of shared office space on Verces, produces a line of specialty toolkits that supplement those presented on page 221 of the Core Rulebook. These toolkits provide circumstance bonuses to one or more skills in specific situations and include the following: tourist’s kit (grants a +4 bonus to Culture checks to recall knowledge about related to a specific planet), <strong>catalyst rig</strong> (increases your effective skill ranks by 2 for the purpose of determining how long it takes you to craft drugs, poisons, or medicinals), influencer kit (grants a +4 bonus to Diplomacy checks made to change attitude for a specific subculture like mercenaries or augmentation enthusiasts), gymnast’s kit (grants a +4 bonus to Acrobatics checks made to tumble), swimmer’s kit (grants a +4 bonus to Athletics checks made to swim), grifter’s kit (grants a +4 bonus to Sleight of Hand checks made to entertain and pick pocket), and tracker’s kit (grants a +4 bonus to Survival checks made to follow tracks).</p>"
```
Comment (LebombJames) [inline src/items/equipment/tool_kit_akinzi_resources_catalyst_rig.json:58]: For the description on these, you can remove all the other toolkit descriptions so it's something like

```
Akinzi Resources, a startup company operating out of shared office space on Verces, produces a line of specialty toolkits that supplement those presented on page 221 of the Core Rulebook. These toolkits provide circumstance bonuses to one or more skills in specific situations and include the following:
Catalyst Rig: ...
```

File: src/items/equipment/tool_kit_akinzi_resources_grifter’s_kit.json [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        "_id": "97fd98e7-d8a1-4210-9d12-60b066a887c8",
+        "name": "Grifter's Kit",
+        "type": "circumstance",
+        "condition": "",
+        "effectType": "skill",
+        "enabled": true,
+        "max": 4,
+        "modifier": "4",
+        "modifierType": "formula",
+        "notes": "to entertain and pick pocket",
```
Comment (LebombJames) [inline src/items/equipment/tool_kit_akinzi_resources_grifter’s_kit.json:76]: Capitalise "to", same goes for any others

File: src/items/equipment/tool_kit_akinzi_resources_grifter’s_kit.json [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    "equippable": false,
+    "equipped": false,
+    "hands": 2,
+    "identified": true,
+    "level": 1,
+    "modifiers": [
+      {
+        "_id": "97fd98e7-d8a1-4210-9d12-60b066a887c8",
+        "name": "Grifter's Kit",
+        "type": "circumstance",
```
Comment (LebombJames) [inline src/items/equipment/tool_kit_akinzi_resources_grifter’s_kit.json:69]: The bonuses on all these kits are untyped, not circumstance

File: src/items/equipment/tool_kit_disguise_kit.json [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+    },
+    "damage": {
+      "parts": []
+    },
+    "description": {
+      "chat": "",
+      "gmnotes": "",
+      "short": "",
+      "unidentified": "",
+      "value": "<p><span>A tool kit is a set of specialized tools and devices not worth purchasing individually, but which as a collection are required for a given task or provide a circumstance bonus to certain skill checks. The types of tool kits are: <strong>disguise kit</strong> (required for Disguise checks to change appearance), engineering kit (Engineering checks without one take a –2 penalty), hacking kit (required for most Computer checks; see page 137 for more information), navigator’s tools (grants a +4 bonus to Survival checks when orienteering), professional’s tools (provides a +4 bonus to Profession checks for one profession), rider’s kit (provides a +4 bonus to Survival checks to ride creatures), and trapsmith’s tools (provides a +4 bonus to Engineering or Mysticism checks to arm or disarm traps).</span></p>"
```
Comment (LebombJames) [inline src/items/equipment/tool_kit_disguise_kit.json:58]: All of these default tool kits are wrapped in a span, get rid of those

File: src/items/equipment/tool_kit_professional's_tools.json [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+    },
+    "equippable": false,
+    "equipped": false,
+    "hands": null,
+    "identified": true,
+    "level": 1,
+    "modifiers": [
+      {
+        "_id": "0b42dd37-e3a8-45ff-b7b7-f5730ca6e982",
+        "name": "Tools modifier",
```
Comment (LebombJames) [inline src/items/equipment/tool_kit_professional's_tools.json:68]: Name the modifier the same as the item name, same for the rider's kit

File: src/items/equipment/tool_kit_xhinti_holdings_linguist’s_kit.json [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
... worth purchasing individually, but which as a collection are required for a given task or provide a circumstance bonus to certain skill checks.</p>\n<p>Xhinti Holdings, which has corporate offices on Salomo Station and factories on @UUID[Compendium.sfrpg.setting.gzXT0xwsj8kQAoj0]{Akiton}, produces a line of specialty tool kits that supplement those presented on page 221 of the Starfinder Core Rulebook. Xhinti tool kits include the following: a climbing kit (grants a +4 bonus to Athletics checks when climbing), demolitionist’s kit (grants a +4 bonus to Engineering checks when arming and disarming explosives), <strong>linguist’s kit</strong> (grants a +4 bonus to Culture checks when deciphering writing), portable weather station (grants a +4 bonus to Survival checks when predicting the weather), starship repair kit (grants a +4 bonus to Engineering checks when repairing the hull of a starship), and survivalist’s kit (grants a +4 bonus to Survival checks when enduring severe weather or living off the land). When using a demolitionist’s kit or starship repair kit for its specialized purpose, you don’t take the –2 penalty to Engineering checks if you don’t have an engineering kit.</p>"
```
Comment (LebombJames) [inline src/items/equipment/tool_kit_xhinti_holdings_linguist’s_kit.json:58]: Is this Akiton link valid? It should link to a journal entry page. If it works, no worries

**Review by LebombJames — CHANGES_REQUESTED** [review]
You can delete the old general "tool kit" item

File: src/items/equipment/tool_kit_akinzi_resources_catalyst_rig.json (hunk 2 above)
Comment (rmorgens) [reply src/items/equipment/tool_kit_akinzi_resources_catalyst_rig.json:58]: I had originally done that and Lankid told me to leave them in.  Will correct

File: src/items/equipment/tool_kit_akinzi_resources_grifter’s_kit.json (hunk 4 above)
Comment (rmorgens) [reply src/items/equipment/tool_kit_akinzi_resources_grifter’s_kit.json:69]: The second sentence of the description on Nethys 'These toolkits provide **circumstance** bonuses to one or more skills in specific situations and include the following:' indicates they are ALL circumstance, not untyped....

File: src/items/equipment/tool_kit_xhinti_holdings_linguist’s_kit.json (hunk 7 above)
Comment (rmorgens) [reply src/items/equipment/tool_kit_xhinti_holdings_linguist’s_kit.json:58]: I didn't touch that part.  If there is a link to Akiton, it existed before my suggested edits

**Comment by LebombJames** [comment]
Looks good now, nice one

**Pull request merged**